src/OpenFeature.Contrib.Providers.Flagd/DependencyInjection/FlagdProviderOptions.cs

```diff
@@ -0,0 +1,64 @@
+using OpenFeature.Contrib.Providers.Flagd;
+
+namespace OpenFeature.DependencyInjection.Providers.Flagd;
+
+/// <summary>
+/// Configuration options for the Flagd provider.
+/// </summary>
+public record FlagdProviderOptions
+{
+    /// <summary>
+    /// Default name for the Flagd provider.
+    /// </summary>
+    public const string DefaultName = "FlagdProvider";
+
+    /// <summary>
+    /// The host for the provider to connect to. Defaults to "localhost".
+    /// </summary>
+    public string Host { get; set; } = "localhost";
+
+    /// <summary>
+    /// The Port property of the config. Defaults to 8013.
+    /// </summary>
+    public int Port { get; set; } = 8013;
+
+    /// <summary>
+    /// Use TLS for communication between the provider and the host. Defaults to false.
+    /// </summary>
+    public bool UseTls { get; set; } = false;
+
+    /// <summary>
+    /// 
+    /// </summary>
+    public bool CacheEnabled { get; set; } = false;
```

Comment:
The summary for `CacheEnabled` is currently empty. Please add a brief description to clarify its purpose.